tests/js/testUtils.js

```diff
@@ -56,4 +56,12 @@ https://github.com/gpii/universal/LICENSE.txt
         });
     };
 
+    gpii.tests.utils.simulateKeyEvent = function (onElement, keyEvent, keyCode) {
```

Comment:
Successfully combined triggerKeyEvent() and simulateKeyEvent(). YAY! Ready for another look.